persistence/src/commonMain/db_user/com/wire/kalium/persistence/ConversationFolders.sq

```diff
@@ -0,0 +1,48 @@
+import com.wire.kalium.persistence.dao.QualifiedIDEntity;
+import com.wire.kalium.persistence.dao.conversation.folder.ConversationFolderTypeEntity;
+
+CREATE TABLE ConversationFolder (
+    id TEXT NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL,
+    folder_type TEXT AS ConversationFolderTypeEntity NOT NULL
+);
+
+CREATE TABLE LabeledConversation (
+      conversation_id TEXT AS QualifiedIDEntity NOT NULL,
+      folder_id TEXT NOT NULL,
+
+      FOREIGN KEY (conversation_id) REFERENCES Conversation(qualified_id) ON DELETE CASCADE ON UPDATE CASCADE,
+      FOREIGN KEY (folder_id) REFERENCES ConversationFolder(id) ON DELETE CASCADE ON UPDATE CASCADE,
+
+      PRIMARY KEY (conversation_id, folder_id)
```

Comment:
this should be the other way around if you want to query conversations by folder